index.html

```diff
@@ -213,5 +213,115 @@ <h1>Skills</h1>
         </div>
       </div>
     </section>
+
+    <section id="contact-form-section">
+      <div class="logoimage">
+        <a href="index.html"
+          ><img
+            class="logoimage"
+            src="images/image_geometry_1(1).svg"
+            alt="My portfolio's logo"
+        /></a>
+      </div>
+      <div>
+        <h1 class="contact name">Get started</h1>
+        <p class="bio form-text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/mgedpadw" method="POST">
+          <div class="form-container">
+            <label for="username"></label>
+            <input
+              type="text"
+              id="username"
+              name="user_name"
+              placeholder="Your name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <div class="form-container">
+            <label for="email"></label>
+            <input
+              type="text"
+              id="email"
+              name="e-mail"
+              placeholder="Your email"
+              required
+            />
+          </div>
+          <div class="form-container">
+            <label for="message">
+              <textarea
+                name="message"
+                id="message"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                placeholder="Enter your message here..."
+              ></textarea>
```

Comment:
- [ ] Your `<textarea>` seems to lack a `required` attribute so I can send the form without inputting any message. Kindly add the required attribute here so forms without messages can not be sent.